src/twisted/internet/test/test_fdset.py

```diff
@@ -115,6 +115,19 @@ def fail():
         # Getting here means the right thing happened probably.
 
 
+    def test_removeReader_selectables_nokey(self):
+        """
+        L{reactor.removeReader()} accepts an L{IReadDescriptor} provider
+        previously passed to C{reactor.addReader()}. Checks if it doesn't
+        raise KeyError exception
+        """
+        reactor, fd, server = self._simpleSetup()
+        reactor.addReader(fd)
+        reactor._selectables = {}
```

Comment:
If I could know about a real problem.. I just see that after some days of work my WebSocket Server just raises the exception above and it stops accepting new connections. So I need to restart it.
I looked at traceback and thought that it would be better to fix in the sources of twisted.
Do you have any ideas about why such error happens?